schemas/add-to-cart-button.schema.json

```
@@ -0,0 +1,76 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema#",
+	"title": "Add To Cart Button",
+	"description": "he add-to-cart-button is a block responsible for adding products in the Minicart (minicart.v2)",
+	"type": "object",
+	"properties": {
+		"props": {
+			"type": "object",
+			"description": "Block's properties",
+			"properties": {
+				"blockClass": {
+					"anyOf": [
+						{ "type": "string" },
+						{ "$ref": "../shared/responsive-values.schema.json" },
+						{ "$ref": "../shared/blockClass.schema.json" }
+					],
+					"description": "Block container class. This prop’s set value functions as a block identifier for CSS customizations.",
+					"default": ""
+				},
+				"onClickBehavior": {
+					"anyOf": [
+						{
+							"type": "string",
+							"enum": [
+								"go-to-product-page",
+								"add-to-cart",
+								"ensure-sku-selection"
+							]
+						}
+					],
```

```suggestion
					"type": "string",
					"enum": [
						"go-to-product-page",
						"add-to-cart",
						"ensure-sku-selection"
					],
```

porque usar o "anyOf" ? acredito que nesses casos apenas o  type e enum sejam melhores